jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/complex/SortingOverviewTests.java

```diff
@@ -0,0 +1,65 @@
+package io.github.epam.angular.tests.elements.complex;
+
+import io.github.com.pages.AngularPage;
+import io.github.epam.TestsInit;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static io.github.com.pages.AngularPage.sortingOverviewSection;
+import static io.github.epam.site.steps.States.shouldBeLoggedIn;
+
+public class SortingOverviewTests extends TestsInit {
+
+    private static final String DESSERT = "Dessert (100g)";
+    private static final String CALORIES = "Calories";
+    private static final String FAT = "Fat (g)";
+    private static final String CARBS = "Carbs (g)";
+    private static final String PROTEIN = "Protein (g)";
+
+    @BeforeMethod(alwaysRun = true)
+    public void before() {
+        AngularPage angularPage = new AngularPage();
```

Comment:
this might be redundant unless there is no some weird reason

Reply:
fixed